### PR TITLE
Use correct headers for the workers

### PR DESCRIPTION
**Pull request by helicopter88**
This allows us to include <klee.h> and other headers in examples or files uploaded by the users

### CODE REVIEW

File: provisioning/hosts [hunk 1]
```diff
@@ -8,4 +8,9 @@ klee.doc.ic.ac.uk:22 ansible_ssh_user=ubuntu
 klee.doc.ic.ac.uk:22 ansible_ssh_user=ubuntu
 
 [workers]
+cloud-vm-45-5.doc.ic.ac.uk:22 ansible_ssh_user=ubuntu
+cloud-vm-45-3.doc.ic.ac.uk:22 ansible_ssh_user=ubuntu
+cloud-vm-45-1.doc.ic.ac.uk:22 ansible_ssh_user=ubuntu
+cloud-vm-45-4.doc.ic.ac.uk:22 ansible_ssh_user=ubuntu
+cloud-vm-47-213.doc.ic.ac.uk:22 ansible_ssh_user=ubuntu
```
Comment (andronat) [inline provisioning/hosts]: 1. ansible_ssh_user must be encrypted in the ansible vault. We must not expose the login user. (it is already exposed though with this PR...)
2. the ssh port must also be encrypted in the ansible vault.
3. we should change the default ssh port as the name of the server is going to be public and leaving the default port on we will have a lot of bots trying to connect. we need a playbook to handle this.
4. the ssh user must have password authentication disabled. only ssh keys must be allowed. do we have an ansible playbook to take care of this?

Comment (andronat) [inline provisioning/hosts]: Here are some links to see examples and use cases:
1. [here](https://dantehranian.wordpress.com/2015/08/17/encrypting-login-credentials-in-ansible-vault/)
2. [here](https://dantehranian.wordpress.com/2015/07/24/managing-secrets-with-ansible-vault-the-missing-guide-part-1-of-2/)
3. [here](https://dantehranian.wordpress.com/2015/07/24/managing-secrets-with-ansible-vault-the-missing-guide-part-2-of-2/)

**Comment by helicopter88** [comment]
Let's get rid of it for now, and think about encrypting login credentials later

File: src/klee_web/worker/processor/klee_run.py [hunk 2]
```diff
@@ -53,7 +53,7 @@ def run_llvm(self):
         code_file = self.runner.DOCKER_CODE_FILE
         object_file = self.runner.DOCKER_OBJECT_FILE
         llvm_command = ['/usr/bin/clang-3.4',
-                        '-I', '/src/klee/include', '-emit-llvm', '-c', '-g',
+                        '-I', '/home/klee/klee_src/include/klee', '-emit-llvm', '-c', '-g',
```
Comment (andronat) [inline src/klee_web/worker/processor/klee_run.py]: Linter is failling. can you move everything a little bit inwards?

**Comment by helicopter88** [comment]
Fixed, and also fixed a test that was failing due to the new header path

**Comment by ccadar** [comment]
I'll merge this reduced PR, but we need to think about how to manage deployment separately.

**Pull request merged**